mappings/net/minecraft/server/chat/ChatFilterer.mapping

```diff
@@ -0,0 +1,59 @@
+CLASS net/minecraft/class_5514 net/minecraft/server/chat/ChatFilterer
+	FIELD field_26823 LOGGER Lorg/apache/logging/log4j/Logger;
+	FIELD field_26824 NEXT_WORKER_ID Ljava/util/concurrent/atomic/AtomicInteger;
+	FIELD field_26825 FILTER_THREAD_FACTORY Ljava/util/concurrent/ThreadFactory;
+	FIELD field_26826 chatEndpoint Ljava/net/URL;
```

Comment:
is endpoint the best name? maybe link or url? something about endpoint doesnt seam right

Reply:
An API endpoint is a pretty standard term